x/concentrated-liquidity/incentives_test.go

```diff
@@ -2944,6 +2977,10 @@ func (s *KeeperTestSuite) TestCreateIncentive() {
 			s.Require().Equal(tc.recordToSet, recordInState)
 			s.Require().Equal(tc.recordToSet, incentiveRecord)
 
+			// Ensure that at least the minimum amount of gas was charged (based on number of existing incentives for current uptime)
+			gasConsumed := s.Ctx.GasMeter().GasConsumed() - existingGasConsumed
+			s.Require().True(gasConsumed >= tc.minimumGasConsumed)
```

Comment:
Any reason you chose >= over an equality check?

Reply:
Yes – there seems to be varying amounts of gas charged based on what the underlying function does, so it seemed cleaner to do a >= check and just ensure that the test cases eclipse any dust amount of gas that's charged instead of hard-coding how much gas is currently being charged by each call.